create_graphics.py

```diff
@@ -494,160 +500,159 @@ def stage_zip_files(tiles, zip_dir):
         old_enough = False
         for fhr in sorted(fcst_hours):
-            if cla.graphic_type == 'enspanel':
+            if cla.graphic_type == "enspanel":
                 # Expand template to create a list of ensemble member files and
                 # check if they exist and that they're old enough
-                grib_paths = []
+                ens_paths = []
                 ens_members = list(range(cla.ens_size))
                 for mem in ens_members:
                     mem_path, mem_old_enough = check_file(cla, fhr, mem=mem)
                     if mem_old_enough:
-                        grib_paths.append(mem_path)
-                    old_enough = len(grib_paths) == cla.ens_size
+                        ens_paths.append(mem_path)
+                    old_enough = len(ens_paths) == cla.ens_size
+                grib_paths = ens_paths
             else:
                 # Only checks existence/age of base file for diffs
                 grib_path, old_enough = pre_proc_grib_files(cla, fhr)
 
             # UPP is most likely done writing if it hasn't written in data_age
             # mins (default is 3 to address most CONUS-sized domains)
             if old_enough:
+                grib_paths.append(grib_path)
                 fcst_hours.remove(fhr)
-                fhr_as_list = [fhr]
             else:
                 if cla.all_leads:
                     # Wait on the missing file for an arbitrary 90% of wait time
-                    if time.time() - timer_end > cla.wait_time * 60 * .9:
-                        print(f"Giving up waiting on {grib_path}. \n",
-                              f"Removing accumulated variables from image list \n",
-                              f"{LOG_BREAK}\n")
+                    if time.time() - timer_end > cla.wait_time * 60 * 0.9:
+                        print(
+                            f"Giving up waiting on {grib_path}. \n",
+                            "Removing accumulated variables from image list \n",
+                            f"{LOG_BREAK}\n",
+                        )
                         remove_accumulated_images(cla)
                         # Explicitly set -all_leads to False
                         cla.all_leads = False
                     else:
                         # Break out of loop, wait for the desired period, and start
                         # back at this forecast hour.
-                        print(f'Waiting for {grib_path} to be available.')
+                        print(f"Waiting for {grib_path} to be available.")
                         break
                 # It's safe to continue on processing the next forecast hour
-                print(f'Cannot find specified file(s), continuing to check on \n \
-                    next forecast hour.')
+                print(
+                    "Cannot find specified file(s), continuing to check on \n \
+                    next forecast hour."
+                )
```

Comment:
```suggestion
                print(
                    "Cannot find specified file(s), continuing to check on \
                    next forecast hour."
                )
```
Is this intentionally breaking a message over two lines? That makes it hard to find the message in logs.